frontend/src/metabase/browse/components/BrowseApp.styled.tsx

```diff
@@ -76,3 +63,8 @@ export const CenteredEmptyState = styled(EmptyState)`
   justify-content: center;
   height: 100%;
 `;
+
+export const LearnAboutDataIcon = styled(Icon)`
+  min-width: 14px;
+  min-height: 14px;
+`;
```

Comment:
Without this, the icon begins to shrink when the viewport is narrow. I don't know why. I saw the same problem, and used the same solution, for the icon in the ModelExplanationBanner.